src/main/java/com/cag/cagbackendapi/services/user/impl/UserService.java

```diff
@@ -36,4 +40,27 @@ public UserResponseDto registerUser(RegisterUserRequestDto registerUserRequestDt
 
         return userDao.saveUser(registerUserRequestDto);
     }
+
+    public UserResponseDto getByUserId(String userId) {
+
+        if(userId == null || userId == "") {
+            throw new BadRequestException("Invalid userId", null);
+        }
+
+        UUID userUUID;
+
+        try {
+            userUUID = UUID.fromString(userId);
+        } catch(Exception ex){
+            throw new BadRequestException("Invalid userId", null);
```

Comment:
In addition to moving the message to the constants file, change the "null" on line 55 to be "ex". The "Invalid User Id" message is the message we send to the user, but the second parameter is the exception that was thrown that we want to log in our app. We'll be able to read the log and see what exception caused the issue if we pass through the "ex" exception.